proto/dropbox/dropbox.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package dropbox;
+
+service Dropbox {
+  rpc GetApproverInfo(GetApproverInfoRequest) returns(GetApproverInfoResponse) {
+  }
+  rpc CanApproveToUser(CanApproveToUserRequest)
+      returns(CanApproveToUserResponse) {}
+  rpc CreateRequest(CreateRequestRequest) returns(CreateRequestResponse) {}
+  rpc HasTransfer(HasTransferRequest) returns(HasTransferResponse) {}
+  rpc GetTransfersInfo(GetTransfersInfoRequest)
+      returns(GetTransfersInfoResponse) {}
```

Comment:
I wouldn't break lines here. But if you do, I think you should do it in all of them.
It looks really messy right now.

Reply:
I blame the prettier

Reply:
💁🏻